### DISCUSSION TITLE
Swagger example default null value

### DISCUSSION THREAD

**Discussion by zawsq** [fastapi/fastapi#11300] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from __future__ import annotations
from fastapi import FastAPI
from pydantic import BaseModel, Field
from typing import Optional


class Test(BaseModel):
    a: str | None = None
    b: Optional[str] = None
    c: str | None = Field(default = None, example = None)


app = FastAPI()


@app.post("/test")
async def f_test(data: Test):
    print(data)
    print(Test().model_dump())
```

### Description

Fastapi data seems to return:

```json
{
  "a": "string",
  "b": "string",
  "c": "string"
}
```

Expected:
In swagger ui
```json
{
  "a": null
  "b": null
  "c": null
}
```
In print 
```json
{
  "a": None 
  "b": None 
  "c": None
}
```

### Operating System

Linux

### FastAPI Version

0.110.0

### Pydantic Version

2.6.4

### Python Version

Python 3.11.8 also tested in Python 3.10

**Comment by JavierSanchezCastro** [comment]
What's the question/problem? Since you're not returning anything, it's the expected behaviour

**Reply by zawsq** [reply]
I'm expecting Swagger to set the default value to null instead of a string, as it doesn't have anything to do with returning anything.

similar issue and discussion:

https://github.com/tiangolo/fastapi/pull/9928#issuecomment-1952225901

https://github.com/tiangolo/fastapi/pull/9928

https://github.com/tiangolo/fastapi/discussions/11131

https://github.com/tiangolo/fastapi/discussions/8401

**Reply by JavierSanchezCastro** [reply]
Oh yes, it's a duplicate of #8401.

I think I saw another PR different from #9928, but I didn't find it

**Comment by YuriiMotov** [comment]
Duplicate of https://github.com/fastapi/fastapi/discussions/8401
